### ISSUE TITLE
[FEATURE] Adding google ads to website

### ISSUE THREAD

**Issue by Palakkgoyal** [narayan954/dummygram#621]
### Description

I think this would be nice to have ads (Not from user's perspective) but, we can show ads between posts. For example, after every 10 posts we can show 1 add.
What do you think? I am not sure about this feature. I can implement this but, it depends on you whether you want to show ads or not.

**Comment by narayan954** [comment]
> ### Description
> I think this would be nice to have ads (Not from user's perspective) but, we can show ads between posts. For example, after every 10 posts we can show 1 add. What do you think? I am not sure about this feature. I can implement this but, it depends on you whether you want to show ads or not.
> 
> ### Screenshots
> _No response_
> 
> ### Additional information
> _No response_

We don't have users other than contributors, so it's hard to say I'd be earning from our project contributors.
But actually I'd wish to learn how to implement them here, so you can go ahead! 
It'd be really interesting to implement it here!

**Comment by narayan954** [comment]
should it be level 2 or 3?

**Comment by Palakkgoyal** [comment]
> > ### Description
> > I think this would be nice to have ads (Not from user's perspective) but, we can show ads between posts. For example, after every 10 posts we can show 1 add. What do you think? I am not sure about this feature. I can implement this but, it depends on you whether you want to show ads or not.
> > ### Screenshots
> > _No response_
> > ### Additional information
> > _No response_
> 
> We don't have users other than contributors, so it's hard to say I'd be earning from our project contributors. But actually I'd wish to learn how to implement them here, so you can go ahead! It'd be really interesting to implement it here!

Yes, it would be interesting and I think there is nothing bad earning from contributors if they also don't have any problem in this.

**Comment by Palakkgoyal** [comment]
> should it be level 2 or 3? adding level 2 label by default

I had never done this before. Let's see if it get harder than expectations then, we will go for level 3.

**Comment by Palakkgoyal** [comment]
I checked about these ads and for them it's necessary for the site to be deployed. I tried to deploy the site on netlify but there come some dependency issues. Have you tried deploying it? and also why the routes are starting from /dummygram. I think it would be better to remove it(As in the end when you will deploy your site there will be nothing on the domain name, user would have to navigate to that particular url) or are there some particular reasons to have them?
